reel-picker/src/main/java/com/innowisegroup/reelpicker/picker/ui/DatePickerFragment.kt

```diff
@@ -32,19 +32,13 @@ internal class DatePickerFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         val view = inflater.inflate(R.layout.item_date_picker, container, false)
+        applyArguments()
 
-        with(requireArguments()) {
-            localDate = getSerializable(LOCAL_DATE) as? LocalDate ?: LocalDate.now()
-            minLocalDate = getSerializable(MIN_LOCAL_DATE) as? LocalDate ?: MIN_DEFAULT_LOCAL_DATE
-            maxLocalDate = getSerializable(MAX_LOCAL_DATE) as? LocalDate ?: MAX_DEFAULT_LOCAL_DATE
-            wrapSelectionWheel = getBoolean(WRAP_SELECTION_BOOLEAN)
-        }
-
-        localDate?.let { refreshDateValue(it) }
+        refreshDateValue(localDate)
 
         if (savedInstanceState != null) {
```

Comment:
This block can lead to erros. Add it before refreshDateValue(localDate). Because by this time value may be changed in if (savedInstanceState != null) {......} Then you can drop refreshDateValue(localDate) from if() statement